src/ast.methods.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * Check this is a call to a valid ast callee name
+ *
+ * @param {object} node
+ * @param {string} calleeName
+ *
+ * @returns {boolean}
+ */
+function isValidCalleeName(node, calleeName) {
+    const calleeNameRegex = new RegExp(`(.?|x|f)${calleeName}`);
+
+    return node.type === 'ExpressionStatement'
+        && node.expression.type === 'CallExpression'
+        && node.expression.callee.type === 'Identifier'
+        && calleeNameRegex.test(node.expression.callee.name.toLowerCase());
+}
+
+/**
+ * Get all the indexes of the callee name
+ *
+ * @param {object} array
+ * @param {string} calleeName
+ *
+ * @returns {number[]}
+ */
+function findCalleeNameIndexes(array, calleeName) {
+    return array.reduce(
+        (array, node, index) => isValidCalleeName(node, calleeName) ? [...array, index] : array,
+        [],
+    );
+}
+
+export {
+    findCalleeNameIndexes,
+    isValidCalleeName
+}
```

Comment:
my preference would be to export functions directly as you can immediately see whether or not they are exported without scrolling to the bottom of the file.

Reply:
Will do that, thanks